src/ox_modules/module-utils.js

```diff
@@ -42,4 +44,64 @@ export default class UtilsModule extends OxygenModule {
         utils.assertArgumentNumberNonNegative(ms, 'ms');
         deasync.sleep(ms);
     }
+
+    /**
+     * @summary Reads data from csv file
+     * @function readCsv
+     * @param {String} filePath - Absolute path to file
+     * @param {Object} options - Options (https://csv.js.org/parse/options/)
+     */
+    readCsv(filePath, options = {}) {
+        const parse = require('csv-parse/lib/sync');
+        var fs = require('fs');
+        var path = require('path');
+
+        var ext = path.extname(filePath);
+        if (ext !== '.csv' && ext !== '.txt') {
+            throw new OxError(errorHelper.errorCode.CSV_ERROR, 'Unsupported file extension: ' + ext);
+        }
+
+        var data;
+        try {
+            data = fs.readFileSync(filePath, 'utf8');
+        } catch (e) {
+            throw new OxError(errorHelper.errorCode.CSV_ERROR, 'Unable to read parameters file: ' + e);
+        }
+
+        // CSVs produced by OS X Excel have header terminated with 0d0d0a
+        data = data.replace(/\r\r/g, '\r');
+
+        // strip BOM for UTF-8
+        var bom = false;
+        // catches 0xEFBBBF (UTF-8 BOM) because the buffer-to-string
+        // conversion translates it to 0xFEFF (UTF-16 BOM)
+        if (data.charCodeAt(0) === 0xFEFF) {
+            bom = true;
+        }
+
+        try {
+            var table = parse(data, { bom: bom, columns: true, trim: true, ...options });
+        } catch (e) {
+            throw new OxError(errorHelper.errorCode.CSV_ERROR, e.message);
+        }
+
+        return table;
+    }
+
+    /**
+     * @summary Writes data into csv file
+     * @function writeCsv
+     * @param {String} filePath - Absolute path to file
+     * @param {Array} data - Absolute path to file
```

Comment:
Is `Array` type correct? Also description is wrong.

Reply:
fixed